client/templates/selectedProduct/selectedProduct.html

```diff
@@ -0,0 +1,30 @@
+<template name="selectedProduct">
+  <h1>Selected Product:</h1>
+    {{#if Template.subscriptionsReady}}
+      {{#if selectedProduct}}
+        {{#with selectedProduct}}
+          {{>product}}
+        {{/with}}
+      {{else}}
+        Sorry, no product with that code...
+      {{/if}}
+    {{else}}
+      Loading...
+    {{/if}}
+</template>
+
+<template name="product">
+  <h2>{{description}}</h2>
+  <p>Price: ${{price}}</p>
+  <p>Cost: ${{cost}}</p>
+  <p>itemLookUpCode: {{itemLookUpCode}}</p>
+  <p>itemType: {{itemType}}</p>
+  <p>Quantity: {{quantity}}</p>
+  <p>reorderPoint: {{reorderPoint}}</p>
+  <p>restockLevel:{{restockLevel}}</p>
+  <p>parentItem: {{parentItem}}</p>
+  <p>extendedDescription: {{extendedDescription}}</p>
+  <p>isActive: {{isActive}}</p>
+  <p>msrp: {{msrp}}</p>
+  <p>createdAt: {{createdAt}}</p>
```

Comment:
ah, i see how the comments work now. They're tied to the line that they are directly below. I thought the point of this page was supposed to display all information about it, including when it was added to the database